.github/workflows/dependabot-automerge.yml

```
@@ -1,6 +1,8 @@
 name: Dependabot Auto-merge
 
-on: pull_request
+# Temporarily disabled until CI is fixed and branch protection is configured
+on: []
```

actionlint (1.7.12)

[error] "on" section should not be empty

(syntax-check)